templates/order/details.html

```diff
@@ -63,35 +63,11 @@ <h2 class="order-details__header">
       {% endif %}
       {% include "order/_ordered_items_table.html" %}
       <div class="order-notes">
-      {% if notes %}
-        <h2 class="order-notes__header">{% trans "Notes" context "Order notes list" %}</h2>
-        {% for note in notes %}
-          <small>
-            {% if note.user %}
-              {% if note.user == user %}
-                {% blocktrans with date=note.date %}
-                  {{ date }} by you
-                {% endblocktrans %}
-              {% elif note.user.is_staff %}
-                {% trans 'staff' context 'Order note author is staff' as note_author_staff %}
-                {% blocktrans with author=note_author_staff date=note.date %}
-                  {{ date }} by <span class="text-muted">{{ author }}</span>
-                {% endblocktrans %}
-              {% else %}
-                {% blocktrans with author=note.user date=note.date %}
-                  {{ date }} by {{ author }}
-                {% endblocktrans %}
-              {% endif %}
-            {% else %}
-              {% blocktrans with date=note.date %}
-                {{ date }} by anonymous customer
-              {% endblocktrans %}
-            {% endif %}
-          </small>
-          <p>{{ note.content }}</p>
-        {% endfor %}
+      {% if order.customer_note %}
+        <h2 class="order-notes__header">{% trans "Customer note" context "Customer order note" %}</h2>
```

Comment:
I would say "Your notes" instead since it is in the customer's panel.

Reply:
Sounds better, just I would rename to "Your note" as it could be at most 1 note